### ISSUE TITLE
Problem upgrading from v1.2.0 to v1.3.0, app is not coming fully up

### ISSUE THREAD

**Issue by dvystrcil** [mealie-recipes/mealie#3263]
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue (title field is above this).
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

I upgraded from v1.2.0 to v1.3.0 using the same process (minus backing up, bad on me) since 1.0-beta. I have had no issues with updating till now. 

- K8s
- Sqlite

The app is not responding:
```
upstream connect error or disconnect/reset before headers. reset reason: remote connection failure, transport failure reason: delayed connect error: 111
```

In the mealie.log I am seeing.

```
INFO: 07-Mar-24 07:16:44        Database connection established.
INFO: 07-Mar-24 07:16:44        Context impl SQLiteImpl.
INFO: 07-Mar-24 07:16:44        Will assume non-transactional DDL.
INFO: 07-Mar-24 07:16:46        Checking for migration data fixes
WARNING: 07-Mar-24 07:16:58     "get_all" method is deprecated; use "page_all" instead
```

After a period of time, the pod will get kicked over and the same errors and logs repeat. I can't find any other clues. 

### Steps to Reproduce

Update my deployment.yaml to use the v1.3.0 tag.

### Please provide relevant logs

```
INFO: 07-Mar-24 07:16:44        Database connection established.
INFO: 07-Mar-24 07:16:44        Context impl SQLiteImpl.
INFO: 07-Mar-24 07:16:44        Will assume non-transactional DDL.
INFO: 07-Mar-24 07:16:46        Checking for migration data fixes
WARNING: 07-Mar-24 07:16:58     "get_all" method is deprecated; use "page_all" instead
```

### Mealie Version

I cannot get to the UI, but I am using the Released v1.3.0

### Deployment

Other (please specify below)

### Additional Deployment Details

K8s home lab.

**Comment by hay-kot** [comment]
Are you sure there aren't any more logs? 

It's very unlikely that those are the only logs from startup. It hasn't started up the web server, nor indicate any error.

**Comment by dvystrcil** [comment]
Are there other log files? I am only seeing the mealie.log

**Comment by boc-the-git** [comment]
Docker has its own logs. I assume k8s should have an equivalent.

**Comment by dvystrcil** [comment]
Yes, and the only additional information there was:
```
        User uid:    1000
        User gid:    1000
 ```
 However, I was able to catch an OOM report from ArgoCD. I increased the resource limits a bit and mealie was able to process ... whatever it needed to process. It's up and running now.
 
 Thank you for your quick responses. The my family is enjoying Mealie and it's helping me a lot with weekly planning and grocery lists. No more guessing what should be for dinner, they can look it up! 😁
